product_docs/docs/pem/8/pem_online_help/08_toc_pem_developer_tools/07_psql_tool.mdx

```diff
@@ -0,0 +1,16 @@
+---
+title: "PSQL Tool"
+---
+
+<div id="psql_tool" class="registered_link"></div>
+
+PSQL tool allows user to connect to PostgreSQL/EDB Advanced server using psql terminal.
+
+-   Open PSQL Tool from Tools menu or PSQL tool button from browser tree or from context menu.
+-   PSQL will connect to the current connected database from browser tree.
+
+<img src="../images/psql_tool.png" class="align-center" alt="PSQL tool window" />
```

Comment:
The syntax for images need to be replaced with ! [Alternate_text](<path_to_image_file>/<image_filename.png>)